recipes/r-patrick/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set version = '0.0.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-patrick
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/patrick_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/patrick/patrick_{{ version }}.tar.gz
+  sha256: 20be5193e72d594c8dfe32e1fdd743c5d2077da940ebf11ba343c41dec97b582
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+  noarch: generic
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-dplyr
+    - r-purrr
+    - r-rlang
+    - r-testthat
+    - r-tibble
+
+  run:
+    - r-base
+    - r-dplyr
+    - r-purrr
+    - r-rlang
+    - r-testthat
+    - r-tibble
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('patrick')"           # [not win]
+    - "\"%R%\" -e \"library('patrick')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://github.com/google/patrick
+
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: This is an extension of the 'testthat' package that lets you add parameters to your
+    unit tests. Parameterized unit tests are often easier to read and more reliable,
+    since they follow the DNRY (do not repeat yourself) rule.
+
+  license_family: APACHE
+
+extra:
+  recipe-maintainers:
+   - michaelquinn32
+
+# The original CRAN metadata for this package was:
+
+# Package: patrick
+# Title: Parameterized Unit Testing
+# Version: 0.0.3
+# Authors@R: person(given = "Michael", family = "Quinn", role = c("aut", "cre"), email = "msquinn@google.com")
+# Description: This is an extension of the 'testthat' package that lets you add parameters to your unit tests. Parameterized unit tests are often easier to read and more reliable, since they follow the DNRY (do not repeat yourself) rule.
+# License: Apache License 2.0
+# URL: https://github.com/google/patrick
+# BugReports: https://github.com/google/patrick/issues
+# Depends: R (>= 3.1)
+# Imports: dplyr, purrr, rlang, testthat, tibble
+# Encoding: UTF-8
+# RoxygenNote: 7.1.1
+# NeedsCompilation: no
+# Packaged: 2020-10-27 18:23:21 UTC; msquinn
+# Author: Michael Quinn [aut, cre]
+# Maintainer: Michael Quinn <msquinn@google.com>
+# Repository: CRAN
+# Date/Publication: 2020-10-27 18:50:03 UTC
+
+# See
+# https://docs.conda.io/projects/conda-build for
+# more information about meta.yaml
```

Comment:
Most of the comments are generated to assist in building a recipe for the package, but otherwise are redundant. Those statements in square brackets are important for the proper functioning of a package. Judging from my experience with Conda packaging, your recipe should look like this. Thank you for putting Patrick into Conda!

```suggestion
{% set version = '0.0.3' %}

{% set posix = 'm2-' if win else '' %}
{% set native = 'm2w64-' if win else '' %}

package:
  name: r-patrick
  version: {{ version|replace("-", "_") }}

source:
  url:
    - {{ cran_mirror }}/src/contrib/patrick_{{ version }}.tar.gz
    - {{ cran_mirror }}/src/contrib/Archive/patrick/patrick_{{ version }}.tar.gz
  sha256: 20be5193e72d594c8dfe32e1fdd743c5d2077da940ebf11ba343c41dec97b582

build:
  merge_build_host: True  # [win]
  number: 0
  noarch: generic

  rpaths:
    - lib/R/lib/
    - lib/

requirements:
  build:
    - {{ posix }}zip               # [win]
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]

  host:
    - r-base
    - r-dplyr
    - r-purrr
    - r-rlang
    - r-testthat
    - r-tibble

  run:
    - r-base
    - r-dplyr
    - r-purrr
    - r-rlang
    - r-testthat
    - r-tibble

test:
  commands:
    - $R -e "library('patrick')"           # [not win]
    - "\"%R%\" -e \"library('patrick')\""  # [win]

about:
  home: https://github.com/google/patrick

  license: Apache-2.0
  license_file: LICENSE
  summary: This is an extension of the 'testthat' package that lets you add parameters to your
    unit tests. Parameterized unit tests are often easier to read and more reliable,
    since they follow the DNRY (do not repeat yourself) rule.

  license_family: APACHE

extra:
  recipe-maintainers:
   - michaelquinn32
```